<?xml version="1.0" ?>

<launch>    

    <arg name="en_sim" default="true" />
    <param name="gazebo"                   value="$(arg en_sim)"     type="bool"/>
    <param name="gazebo_robot_name"        value="robotis_manipulator_h" />

    <param name="offset_table"             value="$(find manipulator_h_manager)/config/offset.yaml"/>
    <param name="robot_file_path"          value="$(find manipulator_h_manager)/config/ROBOTIS_MANIPULATOR_H.robot"/>
    <param name="init_file_path"           value="$(find manipulator_h_manager)/config/dxl_init.yaml"/>
      
    <!-- Manipulator Manager -->
    <node name="manipulator_h_manager" pkg="manipulator_h_manager" type="manipulator_h_manager" output="screen"/>

    <!-- add gui node for test -->
    <node name="manipulator_n_gui" pkg="manipulator_h_gui" type="manipulator_h_gui" output="screen"/>

</launch>

